pkg/vfs/vfs.go

```diff
@@ -63,6 +63,14 @@ var (
 		Help:    "size of write distributions.",
 		Buckets: prometheus.LinearBuckets(4096, 4096, 32),
 	})
+	readCount = prometheus.NewCounter(prometheus.CounterOpts{
```

Comment:
the distribution already has counter for read/write
```
juicefs_fuse_read_size_bytes_count 1
juicefs_fuse_written_size_bytes_count 1
```

Reply:
Same as for the other PR, the metric name doesn't make sense to me (byte count ?).